index.html

```diff
@@ -0,0 +1,138 @@
+                    <a class="nav-link" href="#">Drummers Work Shop</a>
+                </li>
+            </ul>
+        </nav>
+        <nav class="mobile-menu">
+            <div class="mobile-menu-bar" id="humburger-icon">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </nav>
+    </header>
+</body>
+<main>
+    <section class="intro-section" id="home">
+        <h2 class="intro-head">"Drummers Making it Big!"</h2>
+        <h1 class="intro-heading">MUSIC FEST
+            <br>
+            DRUMMERS
+            <br>
+            SUMMIT 2022
+        </h1>
+        <p class="intro-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            Impedit aspernatur deleniti assumenda dignissimos nulla fuga,
+            deserunt magnam? Tenetur vitae nisi optio veritatis numquam neque velit quasi.
+            Doloribus sed ratione nobis.</p>
+            <h3 id="date">2022.10.26(WED) ~ 28(FRI)</h3>
+            <p id="date-location">@ICC Central Business District, Abuja</p>
+    </section>
+
+    <section class="program-info" id="program">
+        <h2 id="main-program">Main Program</h1>
+        <div class="program-bar"></div>
+        <div class="program">
+        <article class="program-list">
+            <img class="program-icon" src="img/program_icon_01.png" alt="drum sticks icon">
+            <h4 class="program-topic">Drum Chops Presentation</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+        </article>
+        <article class="program-list">
+            <img class="program-icon" src="img/music-media.png" alt="drum sticks icon">
+            <h4 class="program-topic">Advertising & Media</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+        </article>
+        <article class="program-list">
+            <img class="program-icon" src="img/program_icon_03.png" alt="drum sticks icon">
+            <h4 class="program-topic">Building Connection</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+        </article>
+        <article class="program-list">
+            <img class="program-icon" src="img/program_icon_02.png" alt="drum sticks icon">
+            <h4 class="program-topic">Drum Brands Presentaion</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+        </article>
+        <article class="program-list">
+            <img class="program-icon" src="img/program_icon_01.png" alt="drum sticks icon">
+            <h4 class="program-topic">Drum Brands</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+        </article>
+        </div>
+        <a class="see-more" href="#">SEE THE WHOLE PROGRAMS</a>
+    </section>
```

Comment:
- [OPTIONAL] Great job here 💯 However, I suggest to update the `SEE THE WHOLE PROGRAMS` button on `mobile version`, it should look like the original design 👍

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/196965099-9a352059-9b48-45c2-bc33-759cec77caf3.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/196965154-cab775a5-7261-44e9-af11-4c9246560c89.PNG" width="500" /> |